examples/memory/dapr_session_example.py

```diff
@@ -0,0 +1,380 @@
+"""
```

Comment:
not sure on the naming conventions of the other files in this dir, but this file is under examples/memory, so I think we can drop the `_example` suffix. Thoughts?